deb/common/control

```diff
@@ -102,6 +102,13 @@ Description: Rootless support for Docker.
   Either VPNKit or slirp4netns (>= 0.4.0) needs to be installed separately.
 Homepage: https://docs.docker.com/engine/security/rootless/
 
+Package: docker-compose-plugin
+Priority: optional
```

Comment:
This is not optional

Reply:
What should it be instead? The dependency is `Enhances: docker-ce-cli`.

Reply:
This comes from https://www.debian.org/doc/debian-policy/ch-archive.html#s-priorities, where the most relevant bit is:

> Most Debian packages will have a priority of `optional`. Priority levels other than `optional` are only used for packages that should be included by default in a standard installation of Debian.

Reply:
(For a repository that isn't the main distribution repository, `Priority:` doesn't really do much at all.)